src/cluster/src/error.rs

```diff
@@ -4,6 +4,7 @@ use serde::export::Formatter;
 use fluvio::FluvioError;
```

Comment:
can you split out helm changes only?  this will make easier to digest changes

Reply:
Is there anything you'd like me to change about one or the other? If you're planning on merging them both right away can we just merge them both here? Otherwise I will spend a lot of time trying to extract the changes

Reply:
add more comments.    There are many question that need to be answered.  The helm changes are straightforward.